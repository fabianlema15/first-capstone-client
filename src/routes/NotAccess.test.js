import React from 'react';
import ReactDOM from 'react-dom';
import NotAccessPage from './NotAccess'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer';

describe('<NotAccessPage />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter>
      <NotAccessPage>
      </NotAccessPage>
    </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('Renders NotAccessPage as expected', () => {
	    const tree = renderer.create(<BrowserRouter>
        <NotAccessPage>
        </NotAccessPage>
      </BrowserRouter>).toJSON();
	    expect(tree).toMatchSnapshot();
	  });

})
