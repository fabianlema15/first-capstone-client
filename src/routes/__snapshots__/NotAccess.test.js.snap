// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotAccessPage /> Renders NotAccessPage as expected 1`] = `
<section>
  <h2>
    No Access
  </h2>
  <p>
    You role does not have access to this page.
  </p>
</section>
`;
